types/hexo-fs/index.d.ts

```diff
@@ -426,11 +426,6 @@ export function writeFile(
  */
 export function writeFileSync(path: string, data: any, options?: string | { encoding?: string | null; mode?: string | number; flag?: string }): void;
 
-// Static classes
-export let Stats: Stats;
```

Comment:
This appears to remove an exported variable from an unrelated module?

Reply:
I can reexport class from `fs` to get the same behaviour as before

Reply:
It is now rebased and fixed